main.js

```diff
@@ -22,34 +30,68 @@ if(fs.existsSync("./secrets.json")) {
 		rl.question("Password: ", function(userpassword) {
 			mc_username = username;
 			mc_password = userpassword;
-			for(var i = 0; i < process.stdout.getWindowSize()[1]; i++) {
-				console.log('\n');  // i know it's not the best way to clear a console but i don't know how to do it
-			}
+			console.clear();
 		});
 	});
 }
 
 webserver.createServer(config.ports.web); // create the webserver
 webserver.password = config.password
+var stoppedByPlayer = false;
+var timedStart;
+var lastQueuePlace;
+var chunkData = [];
+var loginpacket;
+var id;
+var totalWaitTime;
+var starttimestring;
+var playTime;
+var options;
+var doing;
+var calcInterval;
+var authInterval;
+var reconnectinterval;
+webserver.restartQueue = config.reconnect.notConnectedQueueEnd;
+if (config.webserver) {
+	webserver.createServer(config.ports.web); // create the webserver
+	webserver.password = config.password
+}
 webserver.onstart(() => { // set up actions for the webserver
 	startQueuing();
 });
 webserver.onstop(() => {
-	stop();
+	stopQueing();
 });
-
-if (config.openBrowserOnStart) {
-    opn('http://localhost:' + config.ports.web); //open a browser window
+if (config.openBrowserOnStart && config.webserver) {
+	opn('http://localhost:' + config.ports.web); //open a browser window
 }
-
-
 // lets
-let proxyClient; // a reference to the client that is the actual minecraft game
+var proxyClient; // a reference to the client that is the actual minecraft game
```

Comment:
why use ``var`` instead of ``let`` ? As far as I know let is the new better keyword to use. And var should only be used to support old browsers. Since this is not a browser application it might be cleaner to stick with the new keywords.